src/lib/config.ts

```diff
@@ -928,6 +928,16 @@ export class default_config {
      */
     auconcreatecontainer: "true" | "false" = "true"
 
+    /**
+     * Initial urls to navigate to when creating a new tab for a new tab group.
+     */
+    tabgroupnewtaburls = {}
+
+    /**
+     * Whether :tab shows completions for hidden tabs (e.g. tabs in other tab groups).
+     */
+    tabshowhidden: "true" | "false" = "false"
+
```

Comment:
Please add `exaliases` for `tgroup* -> tabgroup*` to make them more discoverable when people type `:tab`. You will have to add each one individually.